pkg/terraform/exec/plugins/vsphereprivate/resource_vsphereprivate_import_ova.go

```diff
@@ -183,11 +183,17 @@ func findImportOvaParams(client *vim25.Client, datacenter, cluster, datastore, n
 	// available for use on the HostSystem we will import the
 	// OVA to.
 	for _, hostObj := range hosts {
-		hostObj.Properties(ctx, hostObj.Reference(), []string{"network", "datastore"}, &hostSystemManagedObject)
-
+		foundDatastore = false
+		foundNetwork = false
+		hostObj.Properties(ctx, hostObj.Reference(), []string{"network", "datastore", "runtime"}, &hostSystemManagedObject)
 		if err != nil {
 			return nil, err
 		}
+
+		// Skip all hosts that are in maintenance mode.
+		if hostSystemManagedObject.Runtime.InMaintenanceMode {
+			break
```

Comment:
This should be `continue`, right?

Reply:
It absolutely should be... what was I thinking? https://github.com/openshift/installer/commit/ceedd7da83a3b2b99d2ece59723871ef5d5656e9